src/solver/simulation/timeseries-numbers.cpp

```diff
@@ -741,7 +763,8 @@ bool TimeSeriesNumbers::Generate(Study& study)
         // Intra-modal TS : draw and store TS numbres
         drawTSnumbersForIntraModal(
           intramodal_draws, isTSintramodal, nbTimeseriesByMode, study.runtime->random);
-        storeTSnumbersForIntraModal(intramodal_draws, isTSintramodal, year, study.areas);
+        storeTSnumbersForIntraModal(
+          intramodal_draws, isTSintramodal, isTSgenerated, nbTimeseriesByMode, year, study.areas);
 
         // NOT intra-modal TS : draw and store TS numbers
         drawAndStoreTSnumbersForNOTintraModal(
```

Comment:
A correction on inter-modal will also have to be made

Reply:
See https://github.com/AntaresSimulatorTeam/Antares_Simulator/pull/465